<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RTGSdeclaration</class>
 <widget class="QWidget" name="RTGSdeclaration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1060</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>481</width>
      <height>23</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>‰∏≠ÂõΩËØÅÂà∏ÁôªËÆ∞ÁªìÁÆóÊúâÈôêË¥£‰ªªÂÖ¨Âè∏‰∏äÊµ∑ÂàÜÂÖ¨Âè∏ÁâàÊùÉÊâÄÊúâCopyright@2007-2018</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/china-logo.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>1920</width>
     <height>1041</height>
    </rect>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/PROPlogo.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>231</width>
      <height>891</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTreeView" name="treeView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>231</width>
        <height>861</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_15">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>231</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>‰∫§Êî∂ÁÆ°ÁêÜ</string>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/image/dashboard.png</normaloff>:/image/dashboard.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>1691</width>
      <height>1041</height>
     </rect>
    </property>
    <widget class="QWidget" name="page_3">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1691</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/deliverytop.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>1691</width>
        <height>23</height>
       </rect>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>115</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/newmessage.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>110</y>
        <width>1241</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ÂΩìÂâçÊ≤°Êúâ‰ø°ÊÅØ</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>129</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;ÂæÆËΩØÈõÖÈªë&quot;;</string>
      </property>
      <property name="text">
       <string>RTGSÂãæÂçïÊåá‰ª§Áî≥Êä•</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>1641</width>
        <height>881</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1651</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 85, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>110</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 10px 5px;
  background-color: #289bd5;
  color: #fff;
  border-radius: 4px;
  padding: 5px 10px;
  border: none;</string>
       </property>
       <property name="text">
        <string>ÈáçÁΩÆ</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>110</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 10px 5px;
  background-color: #289bd5;
  color: #fff;
  border-radius: 4px;
  padding: 5px 10px;
  border: none;</string>
       </property>
       <property name="text">
        <string>Êü•ËØ¢</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>110</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 10px 5px;
  background-color: gray;
  color: #fff;
  border-radius: 4px;
  padding: 5px 10px;
  border: none;</string>
       </property>
       <property name="text">
        <string>Êü•ËØ¢ÂêéÁª≠ËÆ∞ÂΩï</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_12">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>820</y>
         <width>24</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>üûÇ</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_13">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>820</y>
         <width>24</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_14">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>820</y>
         <width>24</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_15">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>820</y>
         <width>24</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>üûÄ</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_16">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>800</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 10px 5px;
  background-color: #289bd5;
  color: #fff;
  border-radius: 4px;
  padding: 5px 10px;
  border: none;</string>
       </property>
       <property name="text">
        <string>ÂãæÂçïÁî≥Êä•</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_17">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>800</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 10px 5px;
  background-color: #289bd5;
  color: #fff;
  border-radius: 4px;
  padding: 5px 10px;
  border: none;</string>
       </property>
       <property name="text">
        <string>ÁªìÊûúÂØºÂá∫</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>-2</x>
         <y>40</y>
         <width>1641</width>
         <height>60</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_7">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:transparent;border-width:0;border-style:outset</string>
          </property>
          <property name="text">
           <string>                             ËµÑÈáëË¥¶Âè∑</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>ÂÖ®ÈÉ®</string>
          </property>
          <item>
           <property name="text">
            <string>ÂÖ®ÈÉ®</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_10">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:transparent;border-width:0;border-style:outset</string>
          </property>
          <property name="text">
           <string>                        ÂãæÂçïÁä∂ÊÄÅ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>Êú™ÂãæÂçï</string>
          </property>
          <item>
           <property name="text">
            <string>Êú™ÂãæÂçï</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ÂÖ®ÈÉ®</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Â∑≤ÂãæÂçï</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Êó†ÈúÄÂãæÂçï</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_11">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:transparent;border-width:0;border-style:outset</string>
          </property>
          <property name="text">
           <string>                ‰∫§Êî∂Áä∂ÊÄÅ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>ÂÖ®ÈÉ®</string>
          </property>
          <item>
           <property name="text">
            <string>ÂÖ®ÈÉ®</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>‰∫§Êî∂‰∏≠</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Êú™‰∫§Êî∂</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>‰∫§Êî∂ÁªìÊùü</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>1440</x>
         <y>90</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 10px 5px;
  background-color: #289bd5;
  color: #fff;
  border-radius: 4px;
  padding: 5px 10px;
  border: none;</string>
       </property>
       <property name="text">
        <string> Â±ïÂºÄ</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>1631</width>
         <height>581</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>820</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>820</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #000000;</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="tabChangesFocus">
        <bool>false</bool>
       </property>
       <property name="undoRedoEnabled">
        <bool>true</bool>
       </property>
       <property name="acceptRichText">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>820</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #000000; border-left-width: 0px;</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="tabChangesFocus">
        <bool>false</bool>
       </property>
       <property name="undoRedoEnabled">
        <bool>true</bool>
       </property>
       <property name="acceptRichText">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4"/>
   </widget>
   <zorder>stackedWidget_2</zorder>
   <zorder>frame</zorder>
   <zorder>stackedWidget</zorder>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 237, 222);</string>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ê¨¢Ëøé‰ΩøÁî®</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Áî®Êà∑‰ª£Á†ÅÔºöXXXXX</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ÂàáÊç¢Áî®Êà∑</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Áî®Êà∑ÈÄÄÂá∫</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Êìç‰ΩúÂëòÔºöXXXX</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_3">
    <property name="geometry">
     <rect>
      <x>1880</x>
      <y>0</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/close.png</normaloff>:/image/close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>21</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_4">
    <property name="geometry">
     <rect>
      <x>1850</x>
      <y>0</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/rewindow.png</normaloff>:/image/rewindow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>21</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_6">
    <property name="geometry">
     <rect>
      <x>1820</x>
      <y>0</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/minit.png</normaloff>:/image/minit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>21</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_5">
    <property name="geometry">
     <rect>
      <x>1790</x>
      <y>0</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="image.qrc">
      <normaloff>:/image/help.png</normaloff>:/image/help.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>21</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1560</x>
      <y>0</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/PROPcombobox.png);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
